src/cadnano_oxDNA.py

```diff
@@ -1055,7 +1064,63 @@ def print_usage():
     basename = os.path.basename(sys.argv[1])
     topology_file = basename + ".top"
     configuration_file = basename + ".oxdna"
-    
+
+    if print_oxview:
+        # Find colors
+        pos_to_color = {}
+        for vh in cadsys.vhelices:
+            for c in vh.stap_colors:
+                vb, color = c
+                pos_to_color[(vh.num, vb)] = color
+        # Create inverse mapping to find pairs by their positions
+        pos_to_id = {}
+        for bid, helix in id_to_pos.items():
+            pos_to_id.setdefault(helix, []).append(bid)
+
+        # Create mapping to find nucleotide by its index
+        id_to_nucleotide = {n.index: n for s in rev_sys._strands for n in s._nucleotides}
+
+        # Find the index of the scaffold strand (there really should be some
+        # easier way of doing this)
+        strands = [id_to_nucleotide[j].strand for i in pos_to_id.values() for j in i]
```

Comment:
As I understand it, `i` here is an iterable. As such, I'd rather call it some other way, since `i` is usually used as an index. I think giving it a more representative name would make the line a bit more readable

Reply:
Yes, technically `i` is a nucleotide.index, but I see what you mean; I changed both iterables to something more readable